From bc99849cf62a9bc4fdd25740cce37accb0a31b6d Mon Sep 17 00:00:00 2001
From: "K.Takata" <kentkt@csc.jp>
Date: Fri, 5 Oct 2018 19:11:43 +0900
Subject: [PATCH 27/28] nsis: Make the 1st section read only

The first section in the uninstaller should be read only.
It should be always performed.
---
 nsis/gvim.nsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/nsis/gvim.nsi b/nsis/gvim.nsi
index b4cd47a76..d9c3b78d8 100644
--- a/nsis/gvim.nsi
+++ b/nsis/gvim.nsi
@@ -848,6 +848,8 @@ FunctionEnd
 
 ##########################################################
 Section "un.$(str_unsection_register)" id_unsection_register
+	SectionIn RO
+
 	# Apparently $INSTDIR is set to the directory where the uninstaller is
 	# created.  Thus the "vim61" directory is included in it.
 	StrCpy $0 "$INSTDIR"
-- 
2.17.0

